usr/local/share/bastille/create.sh

```diff
@@ -174,7 +176,7 @@ ${NAME} {
 
   vnet;
   vnet.interface = e0b_${uniq_epair};
-  exec.prestart += "jib addm ${uniq_epair} ${INTERFACE}";
+  exec.prestart += "jib addm ${uniq_epair} ${bastille_jail_conf_interface}";
```

Comment:
unsure about this change

Reply:
Hi cedwards, this is just the same way how the current `non-vnet` jail rc.conf is applied, i.e. if the user don't have a defined NIC in the `bastille.conf` then the vnet jail always need a NIC defined when creation, if the user had said NIC in the config then, the vnet jail will take that NIC if the user don't specify one in the creation command, however if the user do define a NIC in the vnet jail creation command this NIC in the config is overridden by.

Here is an example of the current `jail.conf` snippet on non-vnet jails:
```
generate_jail_conf() {
    cat << EOF > "${bastille_jail_conf}"
${NAME} {
  devfs_ruleset = 4;
  enforce_statfs = 2;
  exec.clean;
  exec.consolelog = ${bastille_jail_log};
  exec.start = '/bin/sh /etc/rc';
  exec.stop = '/bin/sh /etc/rc.shutdown';
  host.hostname = ${NAME};
  mount.devfs;
  mount.fstab = ${bastille_jail_fstab};
  path = ${bastille_jail_path};
  securelevel = 2;

  interface = ${bastille_jail_conf_interface};
  ${IPX_ADDR} = ${IP};
  ip6 = ${IP6_MODE};
}
EOF
}
```
And this is the code who handle some checks, last statement always obey and set ${INTERFACE} as expected:
```
if [ -z "${bastille_network_loopback}" ] && [ -n "${bastille_network_shared}" ]; then
    local bastille_jail_conf_interface=${bastille_network_shared}
fi
if [ -n "${bastille_network_loopback}" ] && [ -z "${bastille_network_shared}" ]; then
    local bastille_jail_conf_interface=${bastille_network_loopback}
fi
if [ -n "${INTERFACE}" ]; then
    local bastille_jail_conf_interface=${INTERFACE}
fi
```

Regards